scope.go

```diff
@@ -227,6 +262,16 @@ func (s *scope) inc() error {
 			s.clusterUser.name, s.clusterUser.reqPerMin)
 	}
 
+	err2 := s.checkTokenFreePacketSizeRateLimiters()
```

Comment:
why is this call done in `checkTokenFreeRateLimiters()` and not in `inc()`?
This makes the name of the function `checkTokenFreeRateLimiters` misleading

Reply:
I think this relates to the same comment above: The errors are shadowed, so if I change this to seperate calls it would've lead to a slight behaviour difference (Which I wanted to avoid in this PR).